internal/bminventory/inventory.go

```diff
@@ -2062,6 +2066,15 @@ func (b *bareMetalInventory) updateNetworkParams(params installer.UpdateClusterP
 		return common.NewApiError(http.StatusBadRequest, err)
 	}
 
+	validNetworkTypes := []string{models.ClusterNetworkTypeOVNKubernetes, models.ClusterNetworkTypeOpenShiftSDN}
```

Comment:
Please move this validation to the network package - similar to network.VerifyClusterCIDRsNotOverlap and move the testing as well. 
@ori-amizur WDYT?

Reply:
Seems OK